dbms/src/TiDB/Schema/SchemaBuilder.cpp

```diff
@@ -252,9 +252,14 @@ void SchemaBuilder<Getter, NameMapper>::applyDiff(const SchemaDiff & diff)
         }
         else
         {
-            /// The new non-partitioned table will have a new id
+            // Create the new table.
+            // If the new table is a partition table, this will also overwrite
```

Comment:
It may be better to seperate the logical for ActionAlterTablePartitioning and ActionRemovePartitioning

Reply:
For `ActionRemovePartitioning`, it will
* Receive a schema diff, and add the new non-partition table as a table_id in `partition.adding_definitions`. -- TiFlash should add the new table id to mapping and handle the apply snapshot
* Then receive a schema diff to make the new non-partition table as a normal table and remove the old partition-table

So it is the same logic as `ActionAlterTablePartitioning` in tiflash